schedule/virt_autotest/tumbleweed-virt-guest-install.yaml

```diff
@@ -43,26 +42,29 @@ conditional_schedule:
         SKIP_GUEST_INSTALL:
             0:
                 - virt_autotest/unified_guest_installation
+                - virt_autotest/set_config_as_glue
     virtual_network_test:
         ENABLE_VIR_NET:
             1:
-                - virt_autotest/libvirt_virtual_network_init
-                - virt_autotest/libvirt_host_bridge_virtual_network
                 - virt_autotest/libvirt_nated_virtual_network
+                - virt_autotest/libvirt_host_bridge_virtual_network
+            0:
+                - virt_autotest/libvirt_virtual_network_init
     sriov_network_card_pci_passthrough_test:
         ENABLE_SRIOV_NETWORK_CARD_PCI_PASSTHROUGH:
             1:
                 - virt_autotest/sriov_network_card_pci_passthrough
     hotplugging_test:
         ENABLE_HOTPLUGGING:
             1:
-                - virtualization/universal/hotplugging_guest_preparation
```

Comment:
Similar with `libvirt_virtual_network_init.pm`, I guess it is a dependency of hotplugging tests.

Reply:
Sure I will change and add it properly, and ask Leon as well

Reply:
Hotplugging tests are not written by Leon. You can look into the code on your own and ask for help from Wayne, Roy and Tony.

Reply:
Hotplugging tests are not written by Leon. You can look into the code on your own and ask for help from Wayne, Roy and Tony.